lib/src/main/java/com/orange/ods/compose/component/button/OdsButtonsCommon.kt

```diff
@@ -13,25 +13,47 @@ package com.orange.ods.compose.component.button
 import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.size
 import androidx.compose.material.ButtonDefaults
-import androidx.compose.material.Icon
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ImageBitmap
 import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.graphics.vector.ImageVector
+import com.orange.ods.compose.component.content.OdsComponentIcon
 import com.orange.ods.compose.theme.OdsDisplaySurface
 
+
 /**
- * The icon displayed in every type of buttons
- *
- * @param painter Painter of the icon
+ * A button icon in an [OdsButton].
+ * It is non-clickable and no content description is needed cause a button label is always present.
  */
-@Composable
-internal fun ButtonIcon(painter: Painter) {
-    Icon(
-        painter = painter,
-        contentDescription = null,
-        modifier = Modifier.size(ButtonDefaults.IconSize)
-    )
-    Spacer(Modifier.size(ButtonDefaults.IconSpacing))
+class OdsButtonIcon : OdsComponentIcon {
```

Comment:
I wonder if we need this class because it only has one parameter. That's the first API where the icon is not associated with any other parameter (`contentDescription` or `onClick`) so I am not sure what's the best thing to do (use this class or directly a `Painter` / `ImageVector` / `ImageBitmap`).